native/src/widget/checkbox.rs

```diff
@@ -53,11 +56,22 @@ impl<Message, Renderer: self::Renderer> Checkbox<Message, Renderer> {
             is_checked,
             on_toggle: Box::new(f),
             label: String::from(label),
+            size: 20,
             width: Length::Shrink,
+            spacing: 15,
+            text_size: 20,
```

Comment:
Would be really nice to pull these from an underlying renderer.

Reply:
Yes, I think we can add this to the `Renderer` trait using associated constants.